### PR TITLE
thanos: fix GHSA-m425-mq94-257g

### PR DESCRIPTION
**Pull request by hectorj2f**
related to https://github.com/wolfi-dev/os/pull/7931

### CODE REVIEW

File: thanos-0.31.advisories.yaml [hunk 1]
```diff
@@ -48,3 +48,10 @@ advisories:
         type: fixed
         data:
           fixed-version: 0.31.0-r6
+
+  - id: GHSA-m425-mq94-257g
```
Comment (cpanato) [inline thanos-0.31.advisories.yaml:52]: for this we are doing 

```
  - id: CVE-2023-44487
    aliases:
      - GHSA-m425-mq94-257g
```

File: thanos-0.32.advisories.yaml [hunk 2]
```diff
@@ -48,3 +48,10 @@ advisories:
         type: fixed
         data:
           fixed-version: 0.32.4-r4
+
+  - id: GHSA-m425-mq94-257g
```
Comment (cpanato) [inline thanos-0.32.advisories.yaml:52]: same here

**Review by cpanato — CHANGES_REQUESTED** [review]

File: thanos-0.31.advisories.yaml (hunk 1 above)
Comment (hectorj2f) [reply thanos-0.31.advisories.yaml:52]: That is weird, wolfictl didn't gave me a CVE-ID

Comment (cpanato) [reply thanos-0.31.advisories.yaml:52]: that one is weird

Comment (hectorj2f) [reply thanos-0.31.advisories.yaml]: It appeared id: CVE-2023-44487 associated to another alias `GHSA-qppj-fm5r-hxr3` but when looking on the advisories data. You could find this relation 🤔.

Comment (hectorj2f) [reply thanos-0.31.advisories.yaml]: When looking at https://github.com/advisories/GHSA-m425-mq94-257g, it doesn't seem to have one associated to it.

**Review by cpanato — APPROVED** [review]

**Pull request merged**